### DISCUSSION TITLE
2024-08-20 회의록

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#310] [회의록]
#### > 커리어 코칭

<table>
<tr>
<th>시간</th>
<th>이름</th>
</tr>
<tr>
<td>12:30 ~ 12:50</td>
<td>김병준</td>
</tr>
<tr>
<td>18:00 ~ 18:20</td>
<td>오은</td>
</tr>
</table>

### 🔥 개발 관련

- 

### 🔥 기타 내용

-

#### **> 최종 프로젝트 발표 이전 개발 외 티켓 점검**

- **오은** : 발표 대본 작성 및 발표 예행 연습, 더미 데이터 작성
- **김병준** : 리드미/위키 작성, UT 결과보고서 작성
- **김민지** : 시연 영상 촬영
- **곽현정** : UT 결과보고서

#### **> 최종주차 타임라인**

- **8월 19일 월요일 아침** -> 각 기능 담당자 시연하며 설명 및 (치명적인) 버그 리뷰 및 픽스
  - 배포 버그 확인되어 은님 픽스 후 12:00~ 진행
- **8월 19일 월요일 저녁까지** -> 개인별 문서 작성 후 팀 공유 :: 문서 서식은 댓글 참고 (최대한 길고 자세하게)
- **8월 20일 화요일 아침까지** -> PPT 제작 완료
- **8월 20일 화요일 저녁회의** -> 영상 촬영 완료, 대본 작성 완료 -> 최종 예행 연습
- **8월 20일 화요일 심야** -> 리드미 작성 완료
- **8월 21일 수요일** -> 최종발표

#### **> 최종 프로젝트 종료 이후**

- 개인별 개선과제 리팩토링 방식 : 포크 후 리팩토링 권장
- BaaS 마이그레이션 가이드 문서 배포 예정

### 🔥 부제목

- 내용

## 🍀 공통 트러블 슈팅

- 내용

// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>

**Comment by Captain-Kim** [comment]
전원 대댓글에 추가 내용 부탁드립니다. (10:00 한)

```
## PPT 추가 내용 (김병준)

### 프로젝트 결과물에 대한 완성도 평가

- 평가 점수 : 10점 만점 중 0점
- 평가 점수 사유 : 

### 프로젝트 종료 후 개인 성과

- 프로젝트의 아키텍처를 볼 수 있게 되었습니다.
```

**Reply by eunohhh** [reply]
## PPT 추가 내용 (오은)

### 프로젝트 결과물에 대한 완성도 평가

- 평가 점수 : 10점 만점 중 5점
- 평가 점수 사유 : 처음에 생각했던 것 만큼 기능을 구현하지 못했고, 컨벤션 측면에서도 부족함이 많습니다.

### 프로젝트 종료 후 개인 성과

- 어느정도 규모가 되는 프로젝트에 익숙해 질 수 있었습니다

**Reply by Kyara0324** [reply]
## PPT 추가 내용 (최유상)

### 프로젝트 결과물에 대한 완성도 평가

- 평가 점수 : 10점 만점 중 10점
- 평가 점수 사유 : 팀원분들이 열심히 노력한 끝에 만들어진  완성도가 높은 프로젝트이기 때문에 만점을 줬습니다.

### 프로젝트 종료 후 개인 성과

- tailwind css를 조금이나마 익숙하게 다룰 수 있게 되었습니다.

**Reply by hyunjeongkwak** [reply]
## PPT 추가 내용 (곽현정)

### 프로젝트 결과물에 대한 완성도 평가

- 평가 점수 : 10점 만점 중 8점
- 평가 점수 사유 : 프로젝트 결과물이 대체로 잘 구현되었다고 생각하나 개선이 필요합니다.

### 프로젝트 종료 후 개인 성과

- 일정 관리, 작업 우선 순위 설정 등 프로젝트 관리를 경험하며 배울 수 있었습니다.

**Reply by slsl2** [reply]
## PPT 추가 내용 (김민지)

### 프로젝트 결과물에 대한 완성도 평가

- 평가 점수 : 10점 만점 중 7점
- 평가 점수 사유 : 기능 구현과 UI, 보이는 결과물은 잘 만들어졌지만 몇몇 부분에서 코드의 개선이 필요합니다.

### 프로젝트 종료 후 개인 성과

- 규모가 있는 프로젝트를 경험해볼 수 있었고, realtime, sql 등 새로운 기술을 배울 수 있었습니다.

**Comment by Captain-Kim** [comment]
## 김병준 추가 내용

### PPT 제작

- [x] : PPT 제작
- [ ] : UT 그래프 이미지로 변경
- [ ] : 팀원 추가 내용 작성
- [ ] : 오탈자 검수
- [ ] : 내용 오류 최종 검수
- [ ] : 텍스트 서식 깨지는 것 잡은 후 PDF 변환

**Comment by slsl2** [comment]
## 시연 영상에서 꼭 보여줬으면 하는 부분, 주의해야 할 점이 있다면 대댓글로 부탁드립니다.

## 꼭 보여줬으면 하는 부분 (많아도 좋습니다!)
- 예) 뭐 눌렀을 때 뭐가 나타나는 것
- 예) 카카오 소셜 로그인 보여주세요
- 예) ㅇㅇ 페이지는 모바일만 보여주세요

## 주의해야 할 점
- 없으면 남기시지 않아도 좋습니다!

**Reply by Captain-Kim** [reply]
### 김병준

#### 꼭 보여줘야 하는 부분

- 글쓰기 과정 (AI 이미지 자동생성으로)
- 남의 프로필 들어가서 팔로잉 토글하면서 카운팅이 올라가고 떨어지는 것
- 로그인을 하지 않은 상태에서 남의 프로필 접속 시 블러 오버레이 모달 출력되는 것
- 내가 만든 여정 / 참여한 여정 말고 -> 찜한 여정만

#### 주의해야 할 점

- 프사 변경은 인밸리데이트가 너무 느려서 이상합니다 시연 안 하는 것이 좋을 듯
- 내가 만든 여정, 참여한 여정은 깜빡임이 있어서 찜한 여정만 보여주는 것이 좋을 듯
